h4task1.py

```diff
@@ -0,0 +1,36 @@
+# Author: Andrey Maiboroda
+# brealxbrealx@gmail.com
+# Homework4
+# task1
+
+# this program generate random number from 1-10, and sent back the result of the user input via a print statement.
+
+# this function is checks valid input from user on an integer and range
+def getFloatInput(text):
+    while True:
+        userInput = input(text)
+        try:
+            result = int(userInput)
+            if result in range(1,11):
+                return result
+            else:
+                print('Sorry, your number not in randge 1-10. Try againe')        
+        except:
+            print('Your input is not valid. Only numbers accepted.')
+
+import random
+
+# generate rendom number from range 1-10
+randNumber = random.randint(1,10)
+
+userNumber = print('Try to guess the hidden number from 1 to 10. Input your number, please.')
+
+while userNumber != randNumber:
+    userNumber = getFloatInput('')
```

Comment:
1. You need to read and validate integer, not float type.
2. You should make the message argument optional, so you don't have to pass empty string every time.